next.config.js

```diff
@@ -60,13 +60,13 @@ const nextConfig = {
 
     config.module.rules.push({
       test: /\.css$/,
-      exclude: /@oacore\/design\/lib/,
+      exclude: /\/design\/lib/,
```

Comment:
What does this mean?

Reply:
We need to load CSS files from @design/library globally. Unfortunately, postcss doesn't support something like this:

```
:global {
	import url('@oacore/design/lib/index.css')
}
```

Therefore I needed to do it on webpack level. But there was a problem when @oacore/design was symlinked because regular expression didn't match.  Does it make sense?

Reply:
I will think about it